osu.Game/Skinning/Editor/SkinSelectionHandler.cs

```diff
@@ -215,14 +229,16 @@ private static void updateDrawablePosition(Drawable drawable, Vector2 screenSpac
                 drawable.Parent.ToLocalSpace(screenSpacePosition) - drawable.AnchorPosition;
         }
 
-        private void applyOrigin(Anchor anchor)
+        private void applyOrigins(Anchor origin)
```

Comment:
Why were these methods renamed?

Reply:
> Why were these methods renamed?

There were two components to the rename. One is that `applyAnchor` was split into `applyFixedAnchor` and `applyClosestAnchor`, the latter of which didn't exist before.

The second is that member functions which loop over the selection (`applyFixedAnchors`, `applyClosestAnchors`, `applyOrigins`) were changed to be plural, and static functions that operate on a single `Drawable` (`getClosestAnchor`, `applyAnchor`) were made singular.